src/site/content/en/web-dev-basics-one/global-and-local-scope/index.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: post
+title: Global and local variable scope
+description: Learn about scope and how it works in JavaScript. 
+date: 2022-04-14
+---
+
+In this article, you'll learn about scope and how it works in JavaScript. 
+
+Scope is a fundamental concept in JavaScript and other programming languages that defines the context in which variables are accessed and used. It becomes more useful and applicable to your code as you continue to learn JavaScript and work more with variables. 
+
+Scope can help you:
+
+-  **Use memory more efficiently:** Scope provides the ability to load variables only when needed. If a variable is out of scope, you don't need to make it available to the code that's currently executing.
+-  **Find and fix bugs more easily:** Isolating variables with local scope makes it easier to troubleshoot bugs in your code because, unlike global variables, you can trust that code from an outside scope can't manipulate locally scoped variables.
+-  **Create small blocks of reusable code:** For example, you can write a [pure function](https://en.wikipedia.org/wiki/Pure_function) that doesn't rely on outside scope. You can easily move such a function elsewhere with minimal changes.
```

Comment:
```suggestion
- **Use memory more efficiently:** Scope provides the ability to load variables only when needed. If a variable is out of scope, you don't need to make it available to the code that's currently executing.
- **Find and fix bugs more easily:** Isolating variables with local scope makes it easier to troubleshoot bugs in your code because, unlike global variables, you can trust that code from an outside scope can't manipulate locally scoped variables.
- **Create small blocks of reusable code:** For example, you can write a [pure function](https://en.wikipedia.org/wiki/Pure_function) that doesn't rely on outside scope. You can easily move such a function elsewhere with minimal changes.
```